run.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
Aldo, this was accidentally committed, if you could revert or remove it please.

Reply:
Ok